script.js

```diff
@@ -0,0 +1,86 @@
+var box = $("td"); //Array used to check for the winning patterns.
+var playerTurn = 0; //Variable to switch player turns after each click.
+var playerChoiceAtEnd; //Variable which will decide if the player wants to play again or not.
+
+
+//Function which is responsible for colouring the boxes according to which players turn it is.
+function clickedBox(){
+    $("td").click(function(){
```

Comment:
If you're adding event listeners, don't do it within another function like this. I would do it like this:

$("td").click(clickedBox);

function clickedBox() {
  // code here
}

So we add a click event listener, give it a function to run when that event is triggered and then define what the function does beneath.